lib/auth.ts

```diff
@@ -52,6 +53,10 @@ async function verifyAuthJwt(key: CryptoKey, jwt: string) {
 
 function resolveCookieDomain(request: Request) {
   if (!isBaseUrlAnIp() || isRunningLocally(request)) {
+    const trimmedRequestURL = request.url.replace('https://', '').replace('http://', '').split(':')[0];
+    if (isCookieDomainAllowed(trimmedRequestURL)) {
+      return trimmedRequestURL;
+    }
```

Comment:
If so, this should be changed as well.

```suggestion
    if (isCookieDomainAllowed(domain)) {
      return domain;
    }
```